webpack/components/extensions/Hosts/ActionsBar/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react';
+import { DropdownItem } from '@patternfly/react-core';
+import { CubeIcon } from '@patternfly/react-icons';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { ForemanHostsIndexActionsBarContext } from 'foremanReact/components/HostsIndex';
+
+const HostActionsBar = () => {
+  const {
+    fetchBulkParams,
+    selectedCount,
+    selectAllMode,
+  } = useContext(ForemanHostsIndexActionsBarContext);
+
+  let href = '';
+  if (selectAllMode) {
+    const query = fetchBulkParams({ selectAllQuery: 'created_at < "1 second ago"' });
+    href = foremanUrl(`/change_host_content_source?search=${query}`);
+  } else if (selectedCount > 0) {
+    href = foremanUrl(`/change_host_content_source?search=${fetchBulkParams({})}`);
+  }
+
+  const title = __('Change content source');
+  return (
+    <>
+      <DropdownItem
+        ouiaId="change-content-s-dropdown-item"
+        key="change-content-source-dropdown-item"
+        title={title}
+        href={href}
+        isDisabled={selectedCount === 0}
+      >
+        <CubeIcon /> {title}
```

Comment:
```suggestion
        isDisabled={selectedCount === 0}
        icon={<CubeIcon />}
      >
        {title}
```

This will actually make it align better (like Delete in this screenshot):

![image](https://github.com/Katello/katello/assets/22042343/8910eeec-51a7-417d-88a7-b3eb5a27a85c)